src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Import/ImportOrchestratorJobTests.cs

```diff
@@ -536,6 +1133,150 @@ public async Task GivenAnOrchestratorJob_WhenCancelledBeforeCompleted_ThenProces
             Assert.True(testQueueClient.JobInfos.All(t => t.Status != JobStatus.Cancelled && !t.CancelRequested));
         }
 
+        private static async Task VerifyJobStatusChangedAsync(int inputFileCount, int concurrentCount, JobStatus jobStatus, int barrier, int succeedCount, int failedCount, int resumeFrom = -1, int completedCount = 0)
```

Comment:
>barrier [](http://example.com/codeflow?start=128&length=7)

Looks like we do not need barrier

Reply:
Ok, I think I can replace barrier with a number.